### ISSUE TITLE
example tour won't build

### ISSUE THREAD

**Issue by tami5** [hecrj/iced#748]
Hey, I've downloaded tour source code with the following Cargo.toml
```rs
[dependencies]
iced = {git = "https://github.com/hecrj/iced", version = "0.2"}
iced_web = "0.3"
env_logger = "0.8"
dotenv = "0.15.0"
```

the error I'm getting

```rs
Container::new(
    |     ^^^^^^^^^^^^^^ the trait `iced_graphics::backend::Image` is not implemented for `iced_wgpu::backend::Backend`

```

Any ideas why I'm facing this issue? 

Thanks

**Comment by tami5** [comment]
Also trying to build it for the web produces everything but when I open the index.html in the browser I get blank page

```make
	rustup run nightly cargo build --package tour --target wasm32-unknown-unknown && wasm-bindgen target/wasm32-unknown-unknown/debug/tour.wasm --out-dir tour --web

```

**Comment by Kaiden42** [comment]
You have to enable the `image` and `debug` feature to be able to compile the tour example.
```Toml
iced = { git = "https://github.com/hecrj/iced", features = ["image", "debug"] }
```

**Comment by tami5** [comment]
thanks a lot @Kaiden42 it did work,

for local however it failed to build or run 

```rs

MESA-INTEL: warning: Performance support disabled, consider sysctl dev.i915.perf_stream_paranoid=0

wgpu error: Validation Error

Caused by:
    In a RenderPass
      note: encoder = `iced_wgpu encoder`
    In a set_scissor_rect command
    Invalid ScissorRect parameters


thread 'main' panicked at 'Handling wgpu errors as fatal by default', /home/tami/.local/share/cargo/registry/src/github.com-1ecc6299db9ec823/wgpu-0.7.0/src/backend/direct.rs:1896:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Any ideas?

**Comment by yusdacra** [comment]
> thanks a lot @Kaiden42 it did work,
> 
> for local however it failed to build or run 
> 
> ```rs
> 
> MESA-INTEL: warning: Performance support disabled, consider sysctl dev.i915.perf_stream_paranoid=0
> 
> wgpu error: Validation Error
> 
> Caused by:
>     In a RenderPass
>       note: encoder = `iced_wgpu encoder`
>     In a set_scissor_rect command
>     Invalid ScissorRect parameters
> 
> 
> thread 'main' panicked at 'Handling wgpu errors as fatal by default', /home/tami/.local/share/cargo/registry/src/github.com-1ecc6299db9ec823/wgpu-0.7.0/src/backend/direct.rs:1896:5
> note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
> ```
> 
> Any ideas?

This looks like the same problem as #738

**Comment by hecrj** [comment]
Fixed by #818.
